<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>612</width>
    <height>465</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Spectral Compressor</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <layout class="QVBoxLayout" name="verticalLayout_7" stretch="2,1">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout" stretch="0,1">
          <item>
           <widget class="QLabel" name="label_4">
            <property name="text">
             <string>Original View</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QGLViewer" name="origViewer"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_4" stretch="0,1">
          <item>
           <widget class="QLabel" name="label_5">
            <property name="text">
             <string>Compressed View</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QGLViewer" name="compViewer"/>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox_2">
        <property name="title">
         <string>Codec Info</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_3">
         <item row="0" column="0">
          <layout class="QHBoxLayout" name="horizontalLayout_6">
           <item>
            <widget class="QLabel" name="lblMeshName">
             <property name="text">
              <string>&lt;b&gt;No mesh loaded&lt;/b&gt;</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="lblMeshLoadTime">
             <property name="text">
              <string>Loaded in &lt;b&gt;0 ms&lt;/b&gt;</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="1" column="0">
          <layout class="QHBoxLayout" name="horizontalLayout_7">
           <item>
            <widget class="QLabel" name="lblMeshCompTime">
             <property name="text">
              <string>encoded in &lt;b&gt;0 ms&lt;/b&gt;</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="lblMeshDecompTime">
             <property name="text">
              <string>decoded in &lt;b&gt;0 ms&lt;/b&gt;</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="2" column="0">
          <layout class="QHBoxLayout" name="horizontalLayout_9">
           <item>
            <widget class="QLabel" name="lblMeshSize">
             <property name="text">
              <string>filesize: &lt;b&gt;0 KiB&lt;/b&gt;</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="lblMeshCompSize">
             <property name="text">
              <string>compressed: &lt;b&gt;0 KiB&lt;/b&gt;</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="lblCompRatio">
             <property name="text">
              <string>compression ratio: &lt;b&gt;0%&lt;/b&gt;</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="3" column="0">
          <layout class="QHBoxLayout" name="horizontalLayout_8">
           <item>
            <widget class="QLabel" name="lblRmse">
             <property name="toolTip">
              <string>Root Mean Squared Error</string>
             </property>
             <property name="text">
              <string>RMSE: &lt;b&gt;0%&lt;/b&gt;</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </item>
    <item row="0" column="1">
     <layout class="QVBoxLayout" name="verticalLayout_6">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_5">
        <item>
         <widget class="QPushButton" name="btnLoadMesh">
          <property name="text">
           <string>&amp;Load Mesh</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btnCompress">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>&amp;Compress!</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox">
        <property name="title">
         <string>Settings</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_2">
         <item row="2" column="0">
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <item>
            <widget class="QLabel" name="label">
             <property name="text">
              <string>coefficients %</string>
             </property>
            </widget>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_3">
             <item>
              <widget class="QSpinBox" name="spinCoeffs">
               <property name="minimum">
                <number>1</number>
               </property>
               <property name="maximum">
                <number>100</number>
               </property>
               <property name="value">
                <number>100</number>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QSlider" name="slideCoeffs">
               <property name="minimum">
                <number>1</number>
               </property>
               <property name="maximum">
                <number>100</number>
               </property>
               <property name="value">
                <number>100</number>
               </property>
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </item>
         <item row="3" column="0">
          <layout class="QHBoxLayout" name="horizontalLayout_4">
           <item>
            <widget class="QLabel" name="label_2">
             <property name="text">
              <string>quantization bits</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSpinBox" name="spinQuant">
             <property name="toolTip">
              <string>set to 0 to use non-uniform quantization</string>
             </property>
             <property name="minimum">
              <number>0</number>
             </property>
             <property name="maximum">
              <number>31</number>
             </property>
             <property name="value">
              <number>12</number>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="4" column="0">
          <layout class="QVBoxLayout" name="verticalLayout_3">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_5">
             <item>
              <widget class="QLabel" name="label_3">
               <property name="text">
                <string>split at vertex</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QComboBox" name="comboMetis">
               <property name="editable">
                <bool>true</bool>
               </property>
               <item>
                <property name="text">
                 <string>Disabled</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>8</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>16</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>32</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>64</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>128</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>256</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>512</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>1024</string>
                </property>
               </item>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <action name="actionQuit">
   <property name="text">
    <string>&amp;Quit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionOpenMesh">
   <property name="text">
    <string>&amp;Open Mesh</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QGLViewer</class>
   <extends>QWidget</extends>
   <header>QGLViewer/qglviewer.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
